docs/app/layout.js

```diff
@@ -5,7 +5,7 @@ const imgUrl = 'https://repository-images.githubusercontent.com/453236442/aa0db6
 export default function Layout({ children }) {
   return (
     <html>
-      <body>
+      <body suppressHydrationWarning>
```

Comment:
Why we add this, we should need to suppress warning in the boday
```suggestion
      <body>
```

Reply:
I added this because I created an empty Next.js project using `npx create-next-app@latest`, and even after removing most of the content, I still encountered the following error. I believe this is a bug in Next.js, which can lead to our incorrect judgment. Therefore, I have added a prohibition on errors here.

<img width="1586" alt="image" src="https://github.com/huozhi/sugar-high/assets/33956589/61166d9a-f65d-4b8e-a53a-fd102672952b">

<img width="1379" alt="image" src="https://github.com/huozhi/sugar-high/assets/33956589/bc7513eb-fb23-4c0b-9b02-6dcdfcdb871c">

In fact, I'm not very clear whether `suppressHydrationWarning` should really be added here. If you have any knowledge in this area, I look forward to your sharing ❤️

Reply:
Thank you for sharing, I suspect it might be an issue with my Chrome extension. I will roll back this change later.